src/azure-cli/azure/cli/command_modules/databoxedge/__init__.py

```diff
@@ -0,0 +1,46 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+from azure.cli.core import AzCommandsLoader
+import azure.cli.command_modules.databoxedge._help  # pylint: disable=unused-import
+
+
+class DataBoxEdgeManagementClientCommandsLoader(AzCommandsLoader):
+
+    def __init__(self, cli_ctx=None):
+        from azure.cli.core.commands import CliCommandType
+        from .generated._client_factory import cf_databoxedge_cl
```

Comment:
you still want to use generated code, right? 
and we should avoid to use relative import

Reply:
modified